app/src/main/java/nl/mpcjanssen/simpletask/LuaInterpreter.kt

```diff
@@ -129,6 +148,7 @@ object LuaInterpreter {
         fieldTable.set("threshold", dateStringToLuaLong(t.thresholdDate))
         fieldTable.set("createdate", dateStringToLuaLong(t.createDate))
         fieldTable.set("completiondate", dateStringToLuaLong(t.completionDate))
+        fieldTable.set("tasktext", t.taskText)
```

Comment:
I agree. I would have preferred `t.text`, as well, but that method name was already being used (and it returns all tokens), so in order to avoid breaking anything I created `t.tasktext` (which only returns `TToken.TEXT` tokens).

 Redefining or renaming `t.text` is do-able, but seems prone to break things. How would you like this to be handled? e.g. Rename `t.text` to `t.allText` and then rename `t.taskText` to `t.text`?

Reply:
I meant naming the table field text instead of taskText.

Reply:
Renaming the table field `text` instead of `taskText` certainly makes the interface for the user nicer, but I wanted to avoid creating an inconsistency. "threshold" is `t.thresholdDate`, "completiondate" is `t.completionDate`, etc. but "task" would be `t.taskText`, rather than the expected(?) `t.task`. In any case, I'll be glad to change the table field text from "tasktext" to "task".